js/src/modals/CreateAccount/NewGeth/newGeth.js

```diff
@@ -106,7 +106,7 @@ export default class NewGeth extends Component {
     api.parity
       .listGethAccounts()
       .then((_addresses) => {
-        const addresses = (addresses || []).filter((address) => !accounts[address]);
```

Comment:
Yeap. The worst was I tracked this one down in the PRs and it was fixed in master when moved to MobX store - but only after manual testing. I remember this one when I saw it, even there it was also not caught.